packages/dashboard/src/components/iot-dashboard/iot-dashboard.css

```diff
@@ -0,0 +1,25 @@
+:host {
+  display: block;
+}
+
+.container {
+  display: grid;
+  position: absolute;
+  height: 100%;
+  margin: 0.5em;
+}
+.select-rect {
+  border: 2px solid black;
+  position: absolute;
+  background: rgba(0, 0, 1, 0.1);
+}
+
+.grid-image {
+  z-index: -999999999;
+  position: absolute;
+  background-position: top left;
+  background-image: url('./cell.png');
```

Comment:
Was worried about this but saw the stencil config :)

Reply:
Since this is a really simple shape we could alternatively use an SVG here -- would simplify things a bit and we'd be able to edit it as code if we wanted to make changes. Example of how this could work: https://codepen.io/mattpopovich/pen/RwQKeyg/92a6a6ff975906e0e931e0e175e642e8